src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzerResultsTableModel.java

```diff
@@ -262,7 +271,28 @@ public String getType() {
         public Map<String, Object> getCustomStates() {
             return customStates;
         }
-
+        
+        public List<Object> getFormat() {

```

Comment:
Regarding the line, you just need to remove the trailing whitespaces.

The method should be renamed to getValues or something like that, format is not very descriptive.